docs/en/stock-management/stock.settings.md

```diff
@@ -0,0 +1,91 @@
+&raquo; [Home](../index.md) / [Inventory Management](./index.md) / Stock Settings
+
+# Stock Settings
+
+There are a number of settings that affect how the BHIMA software deals with
+stock management.  To check and update the settings, one goes to the "Stock
+Settings" page (under the "Stock" folder):
+
+![Stock Settings Page](./images/stock-settings-page.png)
+
+If you do not see the "Stock Settings" menu item, consult your system
+adminstrator.  You need to be given permission to access that page.
+
+- To add permission for a user to access the "Stock Settings' page: Under
+  `Administration > Role Management`, click on the action menu on the right
+  end of the desired user.  In the action menu, click on `Permission`.  Scroll
+  down to "Stock Settings" (at the end of the Stock items), then click the check
+  box and the [Save] button.  The Stock Settings page should now be accessible
+  for the user.
+
+## AMC - Average Monthly Consumption
+
+For the following Stock Setting options, the term "AMC" is used.  It
+represents "Average Monthy Consumption" of stock.  AMC is the total number of
+stock items dispensed on average to **patients** for the facility involved.
```

Comment:
Let's add "dispensed on average to patients *and services* for the facility involved".  Technically, stock items given to services are assumed to be consumed by patients, but let's make it explicit to avoid future confusion.

Reply:
Done